peachjam/js/utils/function.ts

```diff
@@ -76,18 +76,20 @@ export function generateHtmlTocItems (content: HTMLElement): TOCItemType[] {
       }
       const top = stack[stack.length - 1];
 
-      if (top.type === heading.tagName) {
-        // siblings
-        if (stack.length > 1) {
-          stack[stack.length - 2].children.push(item);
+      if (top) {
```

Comment:
@nickmwangemi Just added this check to prevent the error when the toc is generated